src/components/main/footer/Footer.jsx

```diff
@@ -1,29 +1,30 @@
 import React from 'react';
+import COLORS from '../../../styles/COLORS';
+
+const footerTextStyle = {
+  color: COLORS.BACKGROUND,
+  fontWeight: 'bold',
+  width: '100vw',
+};
 
 const Footer = () => (
-  <footer
-    className="footer"
+  <nav
+    className="navbar has-navbar-fixed-bottom"
     style={{
-      // position: 'fixed',
+      position: 'fixed',
       bottom: '0',
-      width: '100%',
+      height: '45px',
+      background: '#002449',
```

Comment:
Same comment re stripping out inline styles. Probably worthwhile to leave it in for now then refactor when we customize Bulma.

Reply:
Discussed inline styling with @brodly and decided to leave as-is for now.